atomic_reactor/plugins/pre_koji.py

```diff
@@ -45,7 +45,17 @@ def run(self):
             self.log.error("provided target '%s' doesn't exist", self.target)
             raise RuntimeError("Provided target '%s' doesn't exist!" % self.target)
         tag_info = self.xmlrpc.getTag(target_info['build_tag_name'])
+
+        if not tag_info or 'name' not in tag_info.keys():
+            self.log.warning("No tag info was retrieved")
+            return
+
         repo_info = self.xmlrpc.getRepo(tag_info['id'])
+
+        if not repo_info or 'id' not in repo_info.keys():
```

Comment:
I'm not sure, but that's unlikely - the [source code](https://github.com/koji-project/koji/blob/master/koji/tasks.py#L350) shows that its either `None` or a correct dict